### ISSUE TITLE
Clarifications and Suggestions

### ISSUE THREAD

**Issue by Riz-waan** [education/graduation#4445]
I have some questions about this and the rules and would just like some clarification. The context for this is from https://github.com/education/graduation/pull/1382 and https://github.com/education/graduation/pull/4271 but I will try my best to summarize everything onto here. I will say exactly how I felt at some points as it might seem like an overreaction.

### Background Info

I have always been a GitHub user since basically my introduction to Computer Science and programming. In 10th grade when I got the opportunity to teach the rest of my class about programming, I taught them each how to use GitHub. As a high school senior, when I found out my favorite computer science-related company was throwing a graduation event and was giving away swag I was ecstatic. I immediately decided to do as much as I can. I wrote a DEV article. I found a quote. I found the right licensed profile picture and spent some time editing it. Finally, I submitted it way before the deadline.

### The Real Info

I get a notification saying `First Review Completed Successfully`. I was confused and thought something was wrong as it wasn't merged yet, so I happened to ask if everything was good. Within a few days I got the email saying:
> Congratulations on graduating 🎓🎉. Your submission has been approved and you're now part of the class of 2020! We will send you a 📦| ✉️ with swag to your home. Stay tuned for updates through our social media accounts.

And then I received another email saying that they need an image of myself, school emblem, or avatar as "established" on the instructions. It would have been fine if **Mr. Flores** had just said we need a picture of yourself, emblem, avatar without saying *as established on the instructions*. Saying *established* made it seem as though I missed something, so I went back and read all of the instructions and couldn't find anything along those lines so I replied and asked again for a link to the right instructions because I couldn't find them and also asked whether I would be penalized due to lack of supplies for tier 2 swag.

After waiting for about a week, I realized no one is going to look at closed pull requests, so I decided to open a new pull request with the intention of making the changes. This is when **Mr. Shoaib** came to the rescue. He said that he also wasn't aware that you had to have an image of yourself, emblem, avatar as the instructions said profile picture. He also said he thought it was okay as long as an image represents who you are as a person. This is when I realized maybe for next year if you guys decide to do it, to be more clear on the instructions because when I first went through some of the others in the yearbook and I saw pictures that didn't conform to  **Mr. Flores**' "established" instructions I felt as though it was a personal attack or discrimination. But after a bit, I was sure it was probably just a misunderstanding or miscommunication between the reviewers on having one solid rubric per se. I know this seems over-reactive but I have had some religious or racial problems in the past. Also thank you **Mr. Shoaib** for guiding me and telling me to open an issue asking the same.

### Questions

1. Why was my image approved and then reverted by **Mr. Flores** while **Mr. Shoaib** said he would consider it a valid profile picture? *I am sure it could be a misunderstanding between the person writing the instructions and also in between the reviewers.*
2. Why are scenic pictures allowed if they don't clearly follow **Mr. Flores**'s "established" instructions? *This could just be different interpretations of the instructions from different members.*
3. Maybe the next time you guys do this, make the instructions more clear?

Also, I understand you are busy and I would like to say answering this isn't a priority for me or you as it won't change anything for this event, but I would like the answers even if it had to be after the event. Please just don't forget. :)

### Additional Info

I am really really sorry **Mr. Shoaib** and **Mr. Flores** for tagging you guys in my PRs, I had completely misread the instructions. Thank you so much **Mr. Abhishek** *(sorry don't know your last name)* for going out of your way to letting me know and trying to help me out!

I would also take a moment to say that I really appreciate **Mr. Shoaib** for completely going out of his way to help me out!! Thank You!

And thank you so much **Mr. Savani** for supporting and encouraging them in supporting me and clarifying my confusion!

Also, I would like to say that nothing is personal towards you guys and I really hope there are no hard feelings between us. I realize when I am confused and because English is my second language it may seem *argumentative*, *inconsiderate*, or even *provocative*.

I would like to sincerely seek each and every one of your guy's forgiveness for anything I have done. I hope we are completely cool! And sorry for bothering you guys, regardless it is an amazing effort and I appreciate it :)

Sorry for my confusions,

**Rizwaan**

**Comment by juanpflores** [comment]
Hey @Riz-waan 👋,
Nothing to worry about. About your questions:

We wanted to have more of a monotonous experience for the yearbook and that's why we asked for the change of the image. No, it was no personal. We did ask for changes to some submissions. What we looked for was to have some kind of avatar, representation of the school, or avatar to have something similar (or _cooler_) to a yearbook. In the end only, elisemoe and I would have final words on the submissions approvals or changes requested. Though we do appreciate the efforts done by volunteer reviewers 🙌 we never published a rubric for them to follow so many times we had additional changes requested even after volunteers approved them. 

Your submission was successfully approved after you did the change so there's nothing to worry about. 👍
